src/profile/supporting_types/account_or_persona.rs

```diff
@@ -0,0 +1,146 @@
+use crate::prelude::*;
+
+/// Either an `Account` or a `Persona`.
+#[derive(
+    Serialize, Deserialize, Clone, Debug, PartialEq, Hash, Eq, uniffi::Enum,
+)]
+pub enum AccountOrPersona {
+    /// An `Account`
+    ///
+    /// Note:
+    /// This case/variant can not be named `account`/ `Account` due
+    /// to Kotlin UniFFI limitation.
+    AccountEntity(Account),
```

Comment:
If we add post processing we that disambiguates the Account types in Kotlin using full package uri... but at this point in time not worth it